### PR TITLE
{tools}[dummy/] ImageJ v1.51a

### PR DESCRIPTION
**Pull request by sassy-crick**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by sassy-crick** [comment]
Test report by @sassy-crick
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
login002.camp.thecrick.org - Linux RHEL 7.3, Intel(R) Xeon(R) CPU E5-2640 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/85534765a81935c1452f9cbe27e87b7a for a full test report.

File: easybuild/easyconfigs/i/ImageJ/ImageJ-1.51a.eb [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+homepage = 'https://imagej.nih.gov/ij'
+description = "Image Processing and Analysis in Java"
+
+toolchain = {'name': 'dummy', 'version': ''}
+
+source_urls = ['https://imagej.nih.gov/ij/download/src/']
+sources = ['ij%(version_major)s%(version_minor)s-src.zip']
+
+dependencies = [
+    ('Java', '1.8.0_202'),
```
Comment (boegel) [inline easybuild/easyconfigs/i/ImageJ/ImageJ-1.51a.eb]: @sassy-crick Please change this to use the `Java` 1.8 wrapper, now that #7547 is merged?

Comment (sassy-crick) [reply easybuild/easyconfigs/i/ImageJ/ImageJ-1.51a.eb]: @boegel done. I think I forgot to upload that last week.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2016.delcatty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/4546e4eed409f8dd5c2d018a6e0cafae for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2431.golett.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/f8ae4707bedc7ce1a2d7bce5c7fed67d for a full test report.

**Comment by boegel** [comment]
Going in, thanks @sassy-crick!

**Pull request merged**